src/components/InputTooltip.js

```diff
@@ -104,34 +135,45 @@ class InputTooltip extends Component {
                 .then(({ suggestions }) => {
                     this.setState(prevState => ({
                         suggestions,
-                        cachedResults: { ...prevState.cachedResults, [query]: suggestions },
+                        cachedResults: {
+                            ...prevState.cachedResults,
+                            [query]: suggestions,
+                        },
                     }));
-                })
-                .catch(() => this.setState({ isError: true }));
+                });
         }
     }
 
-    accept(url) {
-        const { onAccept } = this.props;
-        const trimmedUrl = url.trim();
-
-        if (this.isValidUrl(trimmedUrl)) {
-            onAccept(trimmedUrl);
-        } else {
-            this.setState({ isLinkInvalid: true });
-        }
+    setSuggestionsListClicked(value) {
+        this.suggestionsListClicked = value;
```

Comment:
Why this does not go to state?

Reply:
setState update state asynchronously, so I think it's safer to set it like that.

Reply:
is there any situation where having this in state may be problematic? what about using setState's callback?